### PR TITLE
added hackernews

### PR DESCRIPTION
**Pull request by liljenstolpe**
I added a hackernews / ycombinator social follow.

### CODE REVIEW

**Comment by liljenstolpe** [comment]
Ok, the hackernews icon is not being desaturated to grey, and is WAY too large.  I'm a bit lost here.  Anyone have a pointer as to what I need to fix?

**Comment by liljenstolpe** [comment]
Another issue which I can fix once I unerstand how to make those changes to _social-following.css - the slack hover color should be #4A154B

File: layouts/partials/svg/hackernews.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="256px" height="256px" viewBox="0 0 256 256" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid">
```
Comment (CJ-Jackson) [inline layouts/partials/svg/hackernews.svg:2]: You might want to check the width and height and compared them to the other file..

**Comment by regisphilibert** [comment]
Hi @liljenstolpe  I recently merged #422 which refactors the way socials are handled on the site. It makes it very easy for users to add social services on their project even if it's not built in ananke. You can read more about it here: https://github.com/theNewDynamic/gohugo-theme-ananke#social-follow--share

If want you can submit a new PR. All you have to do is 
1. add your service to: [func/socials/GetBuiltInServicesDefaults](https://github.com/theNewDynamic/gohugo-theme-ananke/blob/master/layouts/partials/func/socials/GetBuiltInServicesDefaults.html)
2. Add the icon (name must match key from above) to `assets/ananke/socials/

NOTE: You should revise this PR's icon, as it does not aligned with Ananke's other built in social icons.

**Pull request closed without merging**